types/jest/index.d.ts

```diff
@@ -153,9 +153,14 @@ declare namespace jest {
      * for the specified module.
      */
     function setMock<T>(moduleName: string, moduleExports: T): typeof jest;
-    /**
-     * Creates a mock function similar to jest.fn but also tracks calls to object[methodName]
-     */
+     /**
+      * Set the default timeout interval for tests and before/after hooks in milliseconds.
+      * Note: The default timeout interval is 5 seconds if this method is not called.
+      */
+    function setTimeout(timeout: number): typeof jest;
+     /**
+      * Creates a mock function similar to jest.fn but also tracks calls to object[methodName]
+      */
```

Comment:
@totano I meant to ask you to push the indentation one space back to keep it consistent with the rest of the file

```diff
      * for the specified module.
      */
     function setMock<T>(moduleName: string, moduleExports: T): typeof jest;
-    /**
-     * Creates a mock function similar to jest.fn but also tracks calls to object[methodName]
-     */
+    ←/**
+     ←* Set the default timeout interval for tests and before/after hooks in milliseconds.
+     ←* Note: The default timeout interval is 5 seconds if this method is not called.
+     ←*/
+    function setTimeout(timeout: number): typeof jest;
+    ←/**
+     ←* Creates a mock function similar to jest.fn but also tracks calls to object[methodName]
+     ←*/
     function spyOn<T extends {}, M extends keyof T>(object: T, method: M): SpyInstance<T[M]>;
     /**
      * Indicates that the module system should never return a mocked version of
```